phabricator-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsNameForPublicIP": {
```

Comment:
Please call this dnsLabelPrefix to set the right expectations with the users as in #966